static/style.css

```diff
@@ -0,0 +1,78 @@
+.out {
+	color:#AAA;
+}
+.out.background {
+	background-color:#AAA;
+	color: #000;
+}
+.in {
+	color:#0d9200;
+}
+.in.background{
+	background-color:#6cec5f;
+	color: #000;
+}
+.just-left {
+	color:#ffa900;
+}
+.just-left.background{
+	background-color:#ffc149;
+	color: #000;
+}
+.card {
+	margin-bottom: 15px;
+}
+.col-card {
+    padding-right: 10px;
+}
+.btn {
+	t:100%;
+}
+#btnOther {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+body, html {
```

Comment:
nitpick: It might be good to put the `body` and `html` selectors at the top of this file

Reply:
Moved to top